tests/10-EdvClient.spec.js

```diff
@@ -17,12 +16,18 @@ describe('EdvClient', () => {
     await mock.server.shutdown();
   });
 
-  it('validator should throw an error when config is invalid', async () => {
-    const config = {invalid: 'invalid'};
+  it('should throw an error when config is invalid', async () => {
     let result;
     let err;
     try {
-      result = validateSchema({payload: config});
+      result = await EdvClient.createEdv({
+        url: 'http://localhost:9876/edvs',
+        config: {
+          sequence: 0,
+          controller: mock.accountId,
+          invalid: 'invalid'
```

Comment:
Please add a comment that you are intentionally adding an invalid property to the config.  The variable name is a good indicator, but I think it's best to be explicit when we're making negative tests so we're not wondering about what's going on.